src/types.ts

```diff
@@ -51,7 +51,7 @@ export type SetErrorImageFn = (args: SetErrorImageProps) => void;
 export type SetupFn = (attributes: Attributes) => void;
 export type FinallyFn = (attributes: Attributes) => void;
 export type GetObservableFn<E> = (attributes: Attributes) => Observable<E>;
-export type IsBotFn = (navigator?: Navigator) => boolean;
+export type IsBotFn = (navigator?: Navigator, platformId?: object) => boolean;
```

Comment:
I think `platformId` should be non-optional:
```suggestion
export type IsBotFn = (navigator: Navigator | undefined, platformId: Object) => boolean;
```

Reply:
And update `createHooks` to something like:
```ts
export function createHooks<E>(platformId: Object, options?: ModuleOptions<E>): HookSet<any> {
```